src/hooks/useFormJson.tsx

```diff
@@ -181,6 +184,38 @@ function updateFormJsonWithComponents(formJson: OHRIFormSchema, formComponents:
   });
 }
 
+const transformSchema = (formJson: OHRIFormSchema) => {
+  formJson.pages.forEach((page) => {
+    if (page.sections) {
+      let sections = page.sections;
+      sections.forEach((section) => {
+        if (section.questions) {
+          let questions = section.questions;
+          questions.map((question) => {
+            transformQuestion(question);
+          });
+        }
+      });
+    }
+  });
+};
```

Comment:
This should be move to its own file and should be configurable. Transformers could implement some special interface:

```ts

interface FormSchemaTransformer {
   transform: (form: OHRIFormSchema) => OHRIFormSchema;
}

// AFE transformer

const AngularFormTransformer: FormSchemaTransformer = {
  transform: (form) => {
    // do transformations
    return form; 
  }
};

// transformer-form-schema.ts

function transformSchema(form: OHRIFormSchema) {
  const transformers = getRegisteredTransformers();
  transformers.forEach(transformer => {
    form = transformer.transform(form);
  });
  return form;
}
```

Reply:
Done